src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -98,7 +98,11 @@ protected function extractAttributes(object $object, ?string $format = null, arr
             $name = $reflMethod->name;
             $attributeName = null;
 
-            if (str_starts_with($name, 'get') || str_starts_with($name, 'has') || str_starts_with($name, 'can')) {
+            if (
+                str_starts_with($name, 'get')
+                || str_starts_with($name, 'has')
```

Comment:
Same for method named: "hash()" ?

Reply:
I guess one could come up with a list of many method names that would produce the same result. Would it be an option to only take method names into account where the next letter after the prefix is not an uppercase letter?

Reply:
@xabbuh that makes way more sense to me, will update.

Reply:
Updated.